src/table/components/virtualized-table/Totals.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useLayoutEffect, memo, useMemo } from 'react';
+import { VariableSizeList } from 'react-window';
+import { TotalsProps } from './types';
+import { HEADER_HEIGHT, PAGINATION_HEIGHT } from './constants';
+import TotalsCell from './TotalsCell';
+import { getTotalsCellStyle } from '../../utils/styling-utils';
+
+const Totals = (props: TotalsProps) => {
+  const { layout, rect, forwardRef, columnWidth, pageInfo, theme, totals, paginationNeeded } = props;
+  const totalsStyle = useMemo(() => getTotalsCellStyle(layout, theme), [layout, theme.name()]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useLayoutEffect(() => {
+    forwardRef?.current?.resetAfterIndex(0, true);
+    forwardRef?.current?.scrollTo(0);
+  }, [layout, pageInfo, forwardRef, columnWidth]);
+
+  if (!totals.isVisible) {
+    return null;
+  }
+
+  let top = totals.atTop ? HEADER_HEIGHT : rect.height - HEADER_HEIGHT;
+  top -= totals.atBottom && paginationNeeded ? PAGINATION_HEIGHT : 0;
+
+  return (
+    <VariableSizeList
+      ref={forwardRef}
+      layout="horizontal"
+      style={{
+        position: 'sticky',
+        top,
+        left: 0,
+        overflow: 'hidden',
+        backgroundColor: totalsStyle.backgroundColor,
+        borderColor: totalsStyle.borderColor,
+        borderStyle: 'solid',
+        borderWidth: '0px',
```

Comment:
is this needed?

Reply:
At least in Firefox the side that are not explicity set, will get 3px as default

Reply:
okok